packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Expressions/ValueExpressions/TypedValueExpression.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.Generator.CSharp.Expressions
     /// </summary>
     /// <param name="Type">Type expected to be returned by value expression.</param>
     /// <param name="Untyped"></param>
-    public abstract record TypedValueExpression(CSharpType Type, ValueExpression Untyped) : ValueExpression
+    public record TypedValueExpression(CSharpType Type, ValueExpression Untyped) : ValueExpression
```

Comment:
in `autorest.csharp`, this class is not abstract.
It is useful when we want to create an expression with a given type instance, which happens in this PR, therefore the abstract is removed here.